src/components/material/material-buttons/generic/MaterialSecondaryButton.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from "react";
+import { Button } from "../../../Buttons/Button";
+import { ButtonSize } from "../../../../core/utils/types/button";
+
+interface MaterialSecondaryButtonProps {
+  label: string;
+  size: ButtonSize;
+  onClick: () => void;
+  dataCy?: string;
+  ariaDescribedBy: string;
+}
+
+const MaterialSecondaryButton: FC<MaterialSecondaryButtonProps> = ({
+  label,
+  size,
+  onClick,
+  dataCy,
+  ariaDescribedBy
+}) => {
+  // If element is currently focused on, we would like to let users open it using enter
+  const handleKeyUp = (key: string) => {
+    if (key === "Enter") {
+      onClick();
+    }
+  };
+
+  if (size !== "small") {
+    return (
+      <Button
+        label={label}
+        buttonType="none"
+        variant="outline"
+        disabled={false}
+        collapsible={false}
+        size="large"
+        onClick={onClick}
+        dataCy={dataCy}
+        ariaDescribedBy={ariaDescribedBy}
+      />
+    );
+  }
+
+  return (
+    <button
```

Comment:
Can we use our own `<Button />` component here?

Reply:
Yes and no. 😅  This is used to create a button that appears like a link. 
<img width="1474" alt="image" src="https://github.com/user-attachments/assets/25ba3055-92c4-4253-ba3d-cf86e37e0fc3">

We could add a prop for link styling, but since it wouldn't benefit other use cases, it might be better to avoid adding this to the `<Button />` component?